specs/change_password/01_main.rst

```diff
@@ -0,0 +1,157 @@
++-------+-----------------+-----------+------------+
+| issue | title           | author    | created    |
++-------+-----------------+-----------+------------+
+| 74_   | Change password | tschubotz | 2019-01-17 |
++-------+-----------------+-----------+------------+
+
+.. _74: https://github.com/gnosis/safe/issues/74
+
+Change password
+===============
+
+1. `Main`_
+2. `User Interface`_
+3. `External Communication`_
+4. Other_
+
+.. _Main:
+
+.. contents:: Table of Contents
+    :depth: 3
+
+1. Problem Definition
+---------------------
+
+* User wants to change the current app password.
+* Current password was compromised and needs to be changed.
+
+2. Inputs
+-----------
+
+- Current password.
+- New password
+    - Minimum 8 characters (>=8)
+    - At least 1 number
+    - At least 1 letter
+    - No more than 2 identical characters in a row
+      (e.g. 111 is not allowed, 11 is fine)
+    - Maximum lenght?
+- Repeat new password
+    - Needs to match
+
+3. Outputs
+------------
+
+- Changed password
+- Error messages
+  - Wrong current password entered
+  - New password does not meet requirements
+  - Repeated new password does not match.
+
+
+4. Use Case Scenarios
+-----------------------
+
+4.1. Happy Case
+~~~~~~~~~~~~~~~
+
+Preconditions
++++++++++++++
+
+- App is setup with a password
+- User has correct password.
+
+Steps
++++++
+
+1. Open app Settings
+2. Open "Change password"
+3. Enter the current password
+4. Enter valid new password
+5. Repeat new password
+6. Save
+
+Postconditions
+++++++++++++++
+
+- User is on the settings screen.
+- App password is changed.
+- User can unlock the app with the new password.
```

Comment:
Same obviously for confirming transactions, I assume :D

Reply:
Yes, of course. I'll add to make it more specific ;)